src/search/axioms.cc

```diff
@@ -60,23 +61,39 @@ AxiomEvaluator::AxiomEvaluator(const TaskProxy &task_proxy) {
             axiom_literals.emplace_back(var.get_domain_size());
 
         // Initialize rules
+        int count = 0;
+        // Since we are skipping some ops, we cannot access them through their
+        // id position directly
+        unordered_map<int, size_t> axiom_id_to_position;
         for (OperatorProxy axiom : axioms) {
             assert(axiom.get_effects().size() == 1);
             EffectProxy cond_effect = axiom.get_effects()[0];
             FactPair effect = cond_effect.get_fact().get_pair();
             int num_conditions = cond_effect.get_conditions().size();
+            // We ignore dummy rules which have no conditions and set the
+            // variable on its default value.
```

Comment:
on -> to

Reply:
After reading the comment I expected a test that checks whether there are conditions.

Reply:
Agreed with Jendrik, I don't see how this comment matches what follows. Also, I don't think rules without conditions or rules that set the default rule are "dummy rules". A "dummy" something is essentially something that does nothing and serves as a kind of placeholder. This doesn't apply here.